### ISSUE TITLE
[1.21.x] [Neoforge] For some reason, the mod breaks the hopper crafting recipe

### ISSUE THREAD

**Issue by ffuentesm** [MehVahdJukaar/sawmill#53]
![image](https://github.com/user-attachments/assets/71098ec5-a7a1-4ef3-8151-000e6ecbce50)
![image](https://github.com/user-attachments/assets/d61974e6-d7df-486e-9f49-d01a3e9f740d)
![image](https://github.com/user-attachments/assets/9f5ee746-7d3c-4f09-add3-297fa47cdfd6)
Neofeorge 21.1.8
sawmill-1.21-1.5.2-neoforge.jar

For some reason when this mod is on, the hopper recipe no longer works

**Comment by MehVahdJukaar** [comment]
Does it happen with no other mod on?

**Comment by ffuentesm** [comment]
Yes Even when I just have the sawmill mod, it breaks the hopper recipe. I think is something todo with the missing chest, I think.

**Comment by MehVahdJukaar** [comment]
So even with just sawmill and nothing else? Very weird.

**Comment by MehVahdJukaar** [comment]
does this happen with latest version?

**Comment by ffuentesm** [comment]
It still happens 
![image](https://github.com/user-attachments/assets/635990c2-4d4a-4107-8df0-560b3c6c29d7)
Also now, for some reason, it breaks the Functional storage crafting that used moded planks
![image](https://github.com/user-attachments/assets/1a9ebf3f-5545-4272-8ff2-1f2ae25c3bf6)
![image](https://github.com/user-attachments/assets/d2853ffb-c003-42d8-9ca6-9a78469ed832)

**Comment by MehVahdJukaar** [comment]
Reset the mod config
